docker/armorblox/Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+armorblox-sdk = "==0.1.2"
```

Comment:
Is there a reason for setting the version to 0.1.2? 
Locking the version to a specific version number will prevent an automatic update when the package is updated to a more advanced version. someone would need to update the version manually. 

Another option is to set it as follows:
`armorblox-sdk = "*"`
This way the version will be updated automatically. 
@wissamg - what do you think will be better here?

Reply:
@ShacharKidor While we try to maintain backward compatibility for our Python SDK, we do not guarantee it at this point as the SDK is still maturing. That's why we wanted to assure compatibility with a specific version rather than allowing arbitrary upgrades which can end up breaking functionality for existing users. Eventually, we want to move to the wildcard option at least for patch version changes.

Reply:
ok so let's keep it this way :)